force-app/main/default/classes/SOQL.cls

```diff
@@ -74,9 +90,12 @@ public with sharing class SOQL {
      * @return The Integer count of Account records.
      * Update the corresponding test class and testGetAccountCount method to cover this method
      */
-    //[public | private ] [static] data_type method_name (input parameters) {
-    //  The body of the method
-    //}
+    public static Integer getAccountCount(String condition) 
+    {
+        String query = 'SELECT COUNT() FROM Account ' + condition;
+        return (Integer)Database.countQuery(query);
```

Comment:
countQuery is for dynamic sql, you can also just use regular aggregate result and return the count extracted from the first line. This way works too